tools/gen-latex-symbols/data/less.mustache

```diff
@@ -0,0 +1,58 @@
+{{#items}}
+@{{name}}: {{px.offset_x}}, {{px.offset_y}}, {{px.width}}, {{px.height}};
+{{/items}}
+
+.sprite-width(@sprite) {
+  width: extract(@sprite, 3);
+}
+
+.sprite-height(@sprite) {
+  height: extract(@sprite, 4);
+}
+
+.sprite-position(@sprite) {
+  @sprite-offset-x: extract(@sprite, 1);
+  @sprite-offset-y: extract(@sprite, 2);
+  background-position: @sprite-offset-x  @sprite-offset-y;
+}
+
+.sprite(@sprite) {
+  .sprite-position(@sprite);
+  background-repeat: no-repeat;
+  overflow: hidden;
+  display: inline-block;
+  .sprite-width(@sprite);
+  .sprite-height(@sprite);
+}
+
+{{#sprite}}
+{{class}} {
+  margin: 5px;
+  line-height: 30px;
+
+  i {
+    cursor: pointer;
```

Comment:
Will this be repeated for each sprite? If yes then maybe add a common class and define common styles in that class?